gobject-introspection.yaml

```diff
@@ -40,8 +43,9 @@ environment:
       - meson
       - openssf-compiler-options
       - posix-libc-utils
-      - py3-packaging
-      - py3-setuptools
+      - py${{vars.py-version}}-build-base
+      - py${{vars.py-version}}-packaging
+      - py${{vars.py-version}}-setuptools
       - python3
       - python3-dev
```

Comment:
drop `python3` and `python3-dev` here. they're either already covered or potentially bring the wrong thing in.

actually, i think just do `py${{vars.py-version}}-build-base-dev` and you'll get what you need.

Reply:
Ah, I need to one day sit down and just browse through all the packages available for the Python multi-versioned efforts. Forgot about `-build-base-dev`, convenient!